content/en/certificate.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Certificate
+status: Feedback Appreciated
+category: technology
+---
+
+
+## What it is
+
+A certificate — also often referred to as a digital certificate, public key certificate, or SSL certificate — is a technology used to help secure communications over the network. 
+Certificates allow us to know that the particular entity we're communicating with is who they say they are. They also allow us to ensure that our communications are private by encrypting the data we send and receive. 
+
+
+## Problem it addresses
+
+When devices communicate over a network there is no inherent guarantee that a particular device is who it says it is.
+Additionally, we can't guarantee that the traffic between any two devices won't be intercepted by a third party.
+Consequently, any communication can potentially be intercepted, compromising sensitive information like usernames and passwords. 
+
+
+## How it helps
+
+Modern email clients that utilize certificates can notify you if a sender's identity is correct, as will web browsers (notice the little lock in front of the address bar of your web browser).
+Modern email clients will notify you if a sender's identity is correct, as will the browsers (notice the little lock in front of the URL). 
```

Comment:
seems duplicated :)

Reply:
oops, this happens when you mix online editing through GitHub suggestion feature and manual editing ^^. Fixed